src/lib/api/EventAPI.ts

```diff
@@ -20,10 +20,11 @@ import axios from 'axios';
 /**
  * Get a single event by UUID
  * @param uuid Search query uuid
- * @param token Bearer token
+ * @param token Bearer token. Optional, but should be provided if you need to
+ * see attendance codes.
  * @returns Event info
  */
-export const getEvent = async (uuid: UUID, token: string): Promise<PublicEvent> => {
+export const getEvent = async (uuid: UUID, token?: string): Promise<PublicEvent> => {
```

Comment:
Is it worth just removing token and updating everywhere if it's not needed?

Reply:
I was going to, but it turns out the token affects whether you can see the access code for events, so removing it would break the admin event page